config-devel.toml

```diff
@@ -26,7 +26,7 @@ db_driver = "postgres"
 pg_username = "user"
 pg_password = "password"
 pg_host = "localhost"
-pg_port = 55432
+pg_port = 5432
```

Comment:
Why did you change a port here?

Reply:
Because 5432 it's a default port for https://catalog.redhat.com/software/containers/detail/5aa63541ac3db95f196086f1?container-tabs=dockerfile. I assume this config is used for unit and integration tests.

Reply:
Then we would need to update docker-compose.yml for local development as well, not a big deal tho